en/mkdocs.yml

```diff
@@ -53,8 +53,7 @@ nav:
   - Home: index.md
   - Get Started:
       - What is Choreo?: get-started/what-is-choreo.md
-      - Quick Start Guide: get-started/quick-start-guide.md
-      - Get Started: get-started/get-started.md
+      - Quick Start Guide: get-started/get-started.md
```

Comment:
Have we applied a page redirect?

Reply:
Added the new content to the existing Quick Start Guide itself so that no redirection is needed.